### DISCUSSION TITLE
Is it possible to set different nodeOrigin property for different types of node?

### DISCUSSION THREAD

**Discussion by Taichiro-S** [xyflow/xyflow#3085] [Q&A]
When nodeOrigin property of resizable node is changed from default value, there appears to be a gap between the mouse cursor and the actual size of the node. (please check [sandbox](https://codesandbox.io/s/reactflow-resizable-node-u71287?file=/ResizeRotateNode.js))

I want to impelment two types of nodes, one is normal node of which nodeOrigin is set to [0.5, 0.5], and thr other is resizable node of which nodeOrigin is set to [0, 0]. However, It seems that it's not currently possible to change the nodeOrigin property for each type of node.
Would it be possible to provide a way to modify the nodeOrigin property for each node type, or otherwise address this issue?

**Comment by josephclawrence** [comment]
I also have a use case for this.
